contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -1,145 +1,241 @@
+
+Our MIT License isn't just for show. You can read [our source code](https://github.com/PostHog/posthog), raise issues and PRs, and build your own integrations. Our [company handbook](/handbook) is open to everyone, as is [how we pay people](/handbook/people/compensation), [our strategy](/handbook/why-does-posthog-exist), and our [product roadmap](/roadmap). Got a question about anything? Ask in [our online community](/questions).
+
+## Comparing PostHog and Amplitude
+
+### Platform
+
+Entering a credit card unlocks almost all PostHog features and you **only pay for what you use**. You get a generous free allowance _every_ month and you can set billing limits to prevent surprise bills.
+
+<ComparisonTable column1="PostHog" column2="Amplitude">
+  <ComparisonRow column1={true} column2={true} feature="Product analytics" description="Track events and conversion, analyze user behavior" />
+  <ComparisonRow column1="Beta" column2={false} feature="Web analytics" description="Easy to use analytics for marketing websites" />
+  <ComparisonRow column1={true} column2={false} feature="Session replay" description="Watch real users use your product, diagnose bugs" />
+  <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Roll out features safely, toggle features for cohorts or individuals" />
+  <ComparisonRow column1={true} column2={true} feature="Experiments" description="Run tests on new features, optimize conversion funnels" />
+  <ComparisonRow column1={true} column2={false} feature="User surveys" description="Collect and analyze feedback, run NPS and PMF surveys " />
+  <ComparisonRow column1="Beta" column2={true} feature="Customer data platform" description="Sync data with third-party tools" />
+  <ComparisonRow column1="Beta" column2={false} feature="Data warehouse" description="A single source for all your important data" />
+  <ComparisonRow column1={true} column2={false} feature="Heatmaps" description="Visualize where users click in your app or website" />
+  <ComparisonRow column1={true} column2={false} feature="Notebooks" description="Ad-hoc analysis and collaboration" />
+  <ComparisonRow column1={true} column2={false} feature="Dark mode" description="Choose between light and dark UI" />
+  <ComparisonRow column1={true} column2={false} feature="Open source" description="Build your own apps and contribute code" />
+</ComparisonTable>
+
+### Product analytics
+
+There are two key differences here:
+
+1. PostHog supports autocapture, which makes it much fast to implement as you don't to manually instrument every event. Don't want autocapture? Just [turn it off](/docs/product-analytics/autocapture) – we offer the best of both worlds.
+
+2. PostHog has a [SQL-based query builder](/docs/hogql) for power users, which enables you to generate custom data analyses using PostHog event data. In future, our query builder will let you analyze data as you would in a Business Intelligence tool, like Metabase.
```

Comment:
```suggestion
2. PostHog has a [SQL-based query builder](/docs/hogql) for power users, which enables you to generate custom data analyses using PostHog event data. In future, our query builder will let you analyze data as you would in a business intelligence tool, like Metabase.
```
The last sentence makes it unclear that we are doing this in our own product, rather than exporting to metabase.